NEWS.md

```diff
@@ -105,7 +108,7 @@
 ### Broom
   
 - Implemented `tidy()`, `augment()`, and `glance()` from tidyverse/broom for
-  `ml_model_generalized_linear_regression()` and `ml_model_linear_regression()`
+  `ml_model_generalized_linear_regression` and `ml_model_linear_regression`
```

Comment:
Isn't `ml_model_generalized_linear_regression` the model object returned by the function `ml_generalized_linear_regression()`? Or can we also think of the model objects as functions somehow?